lib/ros/be/infra/files/terraform/aws/eks-cluster/main.tf

```diff
@@ -113,7 +114,7 @@ resource "aws_iam_role_policy_attachment" "eks-worker-external-dns" {
 }
 
 resource "null_resource" "k8s-tiller-rbac" {
-  depends_on = [module.eks]
+  #depends_on = [module.eks]
```

Comment:
why we need to comment out this?

Reply:
Temporary commented null-resource k8s-tiller-rbac dependancy on eks module. It caused me circular dependancy with module.eks-cluster.module.eks.aws_launch_configuration.workers when I launched ros be infra apply on existing cluster and it planned to update worker nodes config (volumes 50G -> 100G).
I'm not sure if this dependancy is necessary, as k8s-tiller-rbac triggered when kubeconfig file is ready...Does that implies that cluster is ready as well??
I'll keep it in mind and have a closer look when launching cluster infra from scratch